ide/coqide-server.opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "1.2"
+maintainer: "The Coq development team <coqdev@inria.fr>"
+authors: "The Coq development team, INRIA, CNRS, and contributors."
+homepage: "https://coq.inria.fr/"
+bug-reports: "https://github.com/coq/coq/issues"
+dev-repo: "https://github.com/coq/coq.git"
+license: "LGPL-2.1"
+
+available: [ocaml-version >= "4.02.3"]
```

Comment:
This line must likely be updated. In fact, this is also the case in `coq.opam` and `coqide.opam`.

Reply:
Well, I didn't update this as I am not sure if we want to migrate to OPAM 2 or not.

Reply:
But indeed I guess I'll submit a PR fixing several OPAM issues, for 1.2 for now.

Reply:
So now that you fixed the other opam files in #8650, please fix this one here.

Reply:
Fixed, made this depend on that PR.